src/vcpkg/archives.cpp

```diff
@@ -183,6 +197,15 @@ namespace vcpkg
         Checks::check_exit(VCPKG_LINE_INFO, code == 0, "tar failed while extracting %s", archive);
     }
 
+    void extract_tar_cmake(const Path& cmake_tool, const Path& archive, const Path& to_path)
+    {
+        // Note that CMake's built in tar can extract more archive types than many system tars; e.g. 7z
+        const auto code =
+            cmd_execute(Command{cmake_tool}.string_arg("-E").string_arg("tar").string_arg("xzf").path_arg(archive),
+                        WorkingDirectory{to_path});
+        Checks::check_exit(VCPKG_LINE_INFO, code == 0, "tar failed while extracting %s", archive);
```

Comment:
Should this use localized message?

Reply:
I made everything localized in the other PR but this PR is intentionally the minimal change.